src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const rentForDay = 40;
+
+  if (days >= 7) {
+    return rentForDay * days - 50;
+  }
+
+  if (days >= 3) {
+    return rentForDay * days - 20;
```

Comment:
Similarly, extract '3' and '20' into named constants to improve code readability and maintainability, following checklist item 2.